lib/Elastica/Util.php

```diff
@@ -106,11 +106,19 @@ public static function escapeTerm($term)
     {
         $result = $term;
         // \ escaping has to be first, otherwise escaped later once again
-        $chars = ['\\', '+', '-', '&&', '||', '!', '(', ')', '{', '}', '[', ']', '^', '"', '~', '*', '?', ':', '/', '<', '>'];
+        $escapableChars = ['\\', '+', '-', '&&', '||', '!', '(', ')', '{', '}', '[', ']', '^', '"', '~', '*', '?', ':', '/'];
 
-        foreach ($chars as $char) {
+        foreach ($escapableChars as $char) {
             $result = str_replace($char, '\\'.$char, $result);
         }
+        
+        // < and > cannot be escaped, so they should be removed
+        // @see https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#_reserved_characters
+        $nonEscapableChars = ['<', '>'];
+
+        foreach ($nonEscapableChars as $char) {
```

Comment:
I wonder if we should update line https://github.com/ruflin/Elastica/blob/e47dd0e4832b26127c1275fd7bbe1e3ca6138792/lib/Elastica/Util.php#L77 in some way? Interestingly it already doesn't list < and >.

Reply:
I would drop the list of characters in the description of `replaceBooleanWordsAndEscapeTerm`. My reasoning is that it exposes the internal behaviour of the used submethods. The character list creates maintenance backlog as it is not something that the wrapper method `replaceBooleanWordsAndEscapeTerm` has control over at present.
Yet if you believe it adds value then it needs updating too. Are you looking at any particular wording?

Reply:
+1 on removing it. I'm good with most wordings :-)

Reply:
The method description is updated. I believe we can move this PR forward now.